tests/integration/test_requirements_dev.py

```
@@ -232,39 +280,44 @@ def test_has_appropriate_number_of_packages(self):
     @pytest.fixture
     def requirements(self) -> List[Tuple[str, str]]:
         """Parse and return requirements."""
         return parse_requirements(REQUIREMENTS_FILE)
-    
+
     def test_pyyaml_added(self, requirements: List[Tuple[str, str]]):
         """Test that PyYAML was added as per the diff."""
-        pyyaml_entries = [(pkg, ver) for pkg, ver in requirements if pkg == 'PyYAML']
+        pyyaml_entries = [(pkg, ver) for pkg, ver in requirements if pkg == "PyYAML"]
         assert len(pyyaml_entries) == 1
         pkg, ver = pyyaml_entries[0]
```

Ruff (0.14.13)

Unpacked variable `pkg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)